File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        
        if(s.length() != 4 && s.length() != 6) return false;
        
        int number = 0;
        boolean answer = true;
        
        try{
            number = Integer.parseInt(s);
        }catch(NumberFormatException e){
            answer = false;
        }
        
        return answer;
        
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    
    if(s.length !=4 && s.length !=6) return false;
    
    var str = s.toLowerCase();
    
    if(str.includes("x") || str.includes("e") || str.includes("o") || str.includes("b")) return false;
    
    var answer = true;
    var number = Number(str);
    
    
    if(isNaN(number)){
        answer = false;
    }
    
    return answer;
}